dashboard/Dockerfile

```diff
@@ -20,5 +20,7 @@ RUN cp /opt/murano-dashboard/muranodashboard/local/_50_murano.py /opt/horizon/op
 # config file
 COPY local_settings.py /opt/horizon/openstack_dashboard/local/
 RUN tox -e venv -- python manage.py syncdb --noinput
+COPY base.py /opt/base.py
+RUN cp /opt/base.py .tox/venv/lib/python2.7/site-packages/keystoneclient/auth/identity/base.py
```

Comment:
The problem is that the code is using the internal url of keystone, which is not accesible from dashboard since it is a private IP. This path solve that problem just using public endpoints